cli/src/config-schema.json

```diff
@@ -348,6 +348,14 @@
                         "type": "boolean",
                         "description": "Whether to populate the output file with conflict markers before starting the merge tool. See https://github.com/martinvonz/jj/blob/main/docs/config.md#editing-conflict-markers-with-a-tool-or-a-text-editor",
                         "default": false
+                    },
+                    "mode": {
```

Comment:
nit: should be renamed to `diff-mode` or something because it only applies to `diff-args`.

btw, do you have any opinion about `file-diff-args`/`dir-diff-args` stuff?
https://github.com/martinvonz/jj/issues/3696#issuecomment-2116426056

(Even if we add `file`/`dir-diff-args`, it might make sense to add mode flag or config option. There might be `file-by-file` and `files-all-at-once` modes that share the same `file-diff-args`.)

Reply:
+1 on including `diff` in the setting name.

I would even go further: I think we should avoid non-descript `mode` if we can. Who knows what other settings that could be called "mode" will exist in the future.

Perhaps `diff-dir-mode`? (to be joined by `edit-dir-mode`, etc, in the future). I'd prefer to come up with something even better, but this is what I have now.

Reply:
Done